DESCRIPTION

```diff
@@ -25,6 +25,8 @@ Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.3.3
+Remotes:
+    Nixtla/nixtlar
```

Comment:
Packages from GH are not allowed to be dependencies when uploading the finnts package to cran. If we have to use the current dev version of nixtlar to make it work in finnts, then we should push the nixtlar team to publish to cran.